src/containers/BookForm.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { createBook } from '../actions/index';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+const getCategories = () => {
+  const result = [];
+  for (let i = 0; i < categories.length; i += 1) {
+    result.push(
+      <option key={categories[i]} value={categories[i]}>
+        {categories[i]}
+      </option>,
+    );
+  }
+  return result;
+};
+
+class BookForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
- [ ] Since we are using Redux in this project we must follow the presentational and container components pattern, which basically divide components into the ones that are aware of Redux (container components) and the ones that aren't (presentational components). Please I strongly suggest reading more about this [here](https://redux.js.org/basics/usage-with-react#presentational-and-container-components). Since the Book component is a presentational component, therefore, does not connect with Redux. The `handleBookRemove` (`removeOldBook` in our case) should be implemented in the BookList component and passed to the Book component as a callback prop.
![Screenshot from 2020-10-08 10-35-42](https://user-images.githubusercontent.com/30977845/95465880-0c1b0e00-0952-11eb-9956-35581e5cf484.png)